spec/spec_helper.rb

```diff
@@ -56,5 +56,5 @@ def stub_stripe_methods
   allow_any_instance_of(Donation).to receive(:stripe_create_charge)
 end
 
-ENV['GIVIE_SECRET']="this is the testing secret. it is not very secret"
+ENV['GIVIE_SECRET'] = "2d2dd11b2d4d9c55a38db15ff40ee826"
```

Comment:
Is this a real thing? I would prefer to leave the original value so that we can be sure not to have checked in real tokens

Reply:
the requirements for `ActiveSupport::MessageEncryptor.new` changed, it requires a 32 byte key now. Using the old secret gives a `ArgumentError: key must be 32 bytes`

I can cut down the original string to 32 chars.